scripts/initialise_demo.mjs

```diff
@@ -0,0 +1,289 @@
+
+  try {
+    const result = await fetch(healthEndpoint)
+
+    if (result.ok) {
+      console.log(`Service ${serviceName} is healthy`)
+      return true
+    }
+    const errorBody = await result.text()
+
+    const errorMessage = `Service ${serviceName} is unhealthy. Status: ${result.status}, body ${errorBody}`
+    console.error(errorMessage)
+    return false
+  } catch (e) {
+    const errorMessage = `Service ${serviceName} is unhealthy. Error thrown: ${e?.message}`
+    console.error(errorMessage)
+    return false
+  }
+}
+
+async function getSelfIdentity(port) {
+  const selfEndpoint = `http://localhost:${port}/v1/self`
+  const result = await fetch(selfEndpoint)
+
+  if (!result.ok) {
+    const message = `Unexpected error getting self address: ${result.statusText}`
+    console.error(message)
+    throw new Error(message)
+  }
+
+  const body = await result.json()
+  return body.address
+}
+
+async function setIdentity(address, alias, port) {
+  const setAliasEndpoint = `http://localhost:${port}/v1/members/${address}`
+  const result = await fetch(setAliasEndpoint, {
+    method: 'PUT',
+    headers: {
+      'content-type': 'application/json',
+    },
+    body: JSON.stringify({ alias }),
+  })
+
+  if (!result.ok) {
+    const message = `Error setting alias for address ${address} to ${alias} on port ${port}. Error was ${result.statusText}`
+    console.error(message)
+    throw new Error(message)
+  }
+}
+
+async function setIdentities(identityMap, port) {
+  const { hydrogenSelf, energySelf, regulatorSelf } = identityMap
+  // first set the alias to a random alias as the API returns a 409 if it's already set
+  await Promise.all([
+    setIdentity(hydrogenSelf, randomUUID(), port),
+    setIdentity(energySelf, randomUUID(), port),
+    setIdentity(regulatorSelf, randomUUID(), port),
+  ])
+  // now set the alias to the value we want
+  await Promise.all([
+    setIdentity(hydrogenSelf, hydrogenProducerAlias, port),
+    setIdentity(energySelf, energyProviderAlias, port),
+    setIdentity(regulatorSelf, regulatorAlias, port),
+  ])
+}
+
+async function waitForCertificateState(certificateId, state, port) {
+  const txEndpoint = `http://localhost:${port}/v1/certificate/${certificateId}`
+  for (let timeout = 0; timeout < 20000; timeout += 100) {
+    const result = await fetch(txEndpoint)
+    if (!result.ok && result.status !== 404) {
+      const message = `Error getting certificate ${certificateId} on port ${port}. Error was ${result.statusText}`
+      console.error(message)
+      throw new Error(message)
+    }
+    const body = await result.json()
+    if (body.state === state) {
+      return body
+    }
+    await await delay(100)
+  }
+  const message = `Timeout waiting for state ${state} on certificate ${certificateId} for port ${port}`
+  console.error(message)
+  throw new Error(message)
+}
+
+async function initiateCertificate(hydrogenProducerPort, energyProviderPort) {
```

Comment:
I am getting a message from the Lint:

'energyProviderPort' is declared but its value is never read.

what do you think?

also , a lot of messages about console log that I guess can be muted with

```
/* eslint-disable no-console */
```